pkgs/applications/graphics/gimp/plugins/default.nix

```diff
@@ -224,19 +224,15 @@ lib.makeScope pkgs.newScope (
         Filters/Render/Texture...
       */
       pname = "resynthesizer";
-      version = "2.0.3";
+      version = "0-unstable-2024-02-03";
       buildInputs = with pkgs; [ fftw ];
       nativeBuildInputs = with pkgs; [ autoreconfHook ];
       makeFlags = [ "GIMP_LIBDIR=${placeholder "out"}/${gimp.targetLibDir}" ];
```

Comment:
Is this still needed? Skimming the diff, it might not be https://github.com/bootchk/resynthesizer/compare/master...itr-tert:gimp-resynthesizer-scm:master